docs/querying/functions.md

```diff
@@ -380,15 +380,19 @@ do not show up in the returned vector.
 Similarly, `histogram_stdvar(v instant-vector)` returns the estimated standard
 variance of observations in a native histogram.
 
-## `holt_winters()`
+## `double_exponential_smoothing()`
 
-`holt_winters(v range-vector, sf scalar, tf scalar)` produces a smoothed value
+**This function has to be enabled via the [feature flag](../feature_flags.md#experimental-promql-functions) `--enable-feature=promql-experimental-functions`.**
+
+`double_exponential_smoothing(v range-vector, sf scalar, tf scalar)` produces a smoothed value
 for time series based on the range in `v`. The lower the smoothing factor `sf`,
 the more importance is given to old data. The higher the trend factor `tf`, the
 more trends in the data is considered. Both `sf` and `tf` must be between 0 and
 1.
+For additional details, refer to 
+https://www.itl.nist.gov/div898/handbook/pmc/section4/pmc433.htm.
```

Comment:
Nit: I wouldn't have a literal link in a Markdown document. Let's do it like this:

```suggestion
For details, refer to the [NIST Engineering Statistics Hatndbook](https://www.itl.nist.gov/div898/handbook/pmc/section4/pmc433.htm).
```

It's just occurred to me that it would be helpful to also add a note like this:

This smoothing method is sometimes referred to as “Holt's linear method” or less commonly as “Holt-Winters double exponential smoothing”. The latter is the reason why the `double_exponential_smoothing` function was called `holt_winters` in Prometheus v2, which caused confusion because the name “Holt Winters” is more commonly used for a different smoothing method (“Holt-Winters triple exponential smoothing”).

Reply:
:+1: 
I added some additional explanation, but tried to keep it a bit shorter. wdyt?